.github/workflows/CI.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
       # conda setup requires this special shell
       shell: bash -l {0}
       run: |
-        conda build conda.recipe
+        conda build conda.recipe -c conda-canary
```

Comment:
The package has a very unfortunate set of complete other test packages in its source tree, that conda-build will find